astropy/coordinates/transformations.py

```diff
@@ -347,23 +347,38 @@ def get_transform(self, fromsys, tosys):
             self._composite_cache[fttuple] = comptrans
         return self._composite_cache[fttuple]
 
-    def lookup_name(self, name):
+    def lookup_name(self, name, suppress_error=True):
         """
         Tries to locate the coordinate class with the provided alias.
 
         Parameters
         ----------
         name : str
             The alias to look up.
+        suppress_error : bool (default value is False)
+            True : doesn't raise exception if 'name' isn't found. (returns None)
+            False : raises exception if 'name' isn't found.
 
         Returns
         -------
         coordcls
+        if suppress_error == True:
```

Comment:
> Can you please tell me how can I address the comment?

The easy way:

Just make it text: "The coordinate class corresponding to the ``name``. If no such class exists it returns None if suppress_error is true, otherwise it raises an exception."

The complicated way:

Build the docs locally and check how they look. If they look alright you can post it here as image, if it doesn't look okay, then change it _until_ it looks good. :)